src/config/graphical-session/gnome.md

```diff
@@ -27,3 +27,8 @@ The `gdm` package provides the `gdm` service for the GNOME Display Manager;
 [test the service](../services/index.md#testing-services) before enabling it.
 GDM defaults to providing a Wayland session via the `mutter` window manager, but
 an X session can be chosen instead.
+
+## Steps for a working gnome session
+1. install gnome(dbus, gdm, elogind)
+2. enable `dbus` and `elogind`
```

Comment:
dbus is mentioned in Pre-installation, elogind might not be required because it might be started by dbus, did you have an issue with this?

Reply:
Every time I try install gnome without elogind I revive the withe screen of error.

After some tries in different combination, this is the solution I found out to work

Reply:
i think elogind installation should be mentioned in pre-installation

Reply:
Installing elogind is definitely required, not so sure about enabling the service, could you link https://docs.voidlinux.org/config/session-management.html#elogind in Pre-installation?

Reply:
Okkk, after dinner I rewrite the pre-install stuf.

Thanks <3

Reply:
bump